src/components/pages/personal-teacher-page/personal-teacher-tabs/PersonalTeacherTabs.tsx

```diff
@@ -1,216 +1,63 @@
 import React, { FC, useState } from 'react';
-import Link from 'next/link';
-import { useRouter } from 'next/router';
+import { Box } from '@mui/material';
 
-import { SubjectCard } from '@/components/common/composite/cards/subject-card';
-import Button, {
-  ButtonColor,
-  ButtonSize,
-  ButtonVariant,
-} from '@/components/common/ui/button';
-import {
-  TabItem,
-  TabItemContentPosition,
-  TabItemContentSize,
-} from '@/components/common/ui/tab/tab-item/TabItem';
-import { TabList } from '@/components/common/ui/tab/tab-list/TabList';
-import { TabPanel } from '@/components/common/ui/tab/tab-panel/TabPanel';
-import { TabPanelsList } from '@/components/common/ui/tab/tab-panels-list/TabPanelsList';
+import Tab from '@/components/common/ui/tab-mui/tab'; //todo dynamic importats
+import TabContext from '@/components/common/ui/tab-mui/tab-context';
+import TabList from '@/components/common/ui/tab-mui/tab-list';
+import TabPanel from '@/components/common/ui/tab-mui/tab-panel';
+import CommentTab from '@/components/pages/personal-teacher-page/personal-teacher-tabs/components/comment-tab';
+import GeneralTab from '@/components/pages/personal-teacher-page/personal-teacher-tabs/components/general-tab';
+import PollButtons from '@/components/pages/personal-teacher-page/personal-teacher-tabs/components/poll-buttons';
+import SubjectTab from '@/components/pages/personal-teacher-page/personal-teacher-tabs/components/subject-tab';
+import { GetTeacherResponse } from '@/lib/services/teacher/TeacherService';
 
-import styles from './PersonalTeacherTabs.module.scss';
-export type PersonalTeacherTabsProps = {
-  id: string;
-  subjects: {
-    id: string;
-    name: string;
-  }[];
-};
+import * as stylesMUI from './PersonalTeacherTabs.styles';
 
-const TabsPage: FC<PersonalTeacherTabsProps> = props => {
+const PersonalTeacherTabs: FC<GetTeacherResponse> = props => {
```

Comment:
use destructuring to get needed elements from props
```suggestion
const PersonalTeacherTabs: FC<GetTeacherResponse> = ({ oneField, secondField }) => {
```